codegen.mts

```diff
@@ -1,10 +1,143 @@
+          SchemaExplorer: '../shared/mappers#SchemaExplorerMapper',
+          UnusedSchemaExplorer: '../shared/mappers#UnusedSchemaExplorerMapper',
+          DeprecatedSchemaExplorer: '../shared/mappers#DeprecatedSchemaExplorerMapper',
+          GraphQLObjectType: '../shared/mappers#GraphQLObjectTypeMapper',
+          GraphQLInterfaceType: '../shared/mappers#GraphQLInterfaceTypeMapper',
+          GraphQLUnionType: '../shared/mappers#GraphQLUnionTypeMapper',
+          GraphQLEnumType: '../shared/mappers#GraphQLEnumTypeMapper',
+          GraphQLInputObjectType: '../shared/mappers#GraphQLInputObjectTypeMapper',
+          GraphQLScalarType: '../shared/mappers#GraphQLScalarTypeMapper',
+          GraphQLUnionTypeMember: '../shared/mappers#GraphQLUnionTypeMemberMapper',
+          GraphQLEnumValue: '../shared/mappers#GraphQLEnumValueMapper',
+          GraphQLField: '../shared/mappers#GraphQLFieldMapper',
+          GraphQLInputField: '../shared/mappers#GraphQLInputFieldMapper',
+          GraphQLArgument: '../shared/mappers#GraphQLArgumentMapper',
+          OrganizationInvitation:
+            '../shared/entities#OrganizationInvitation as OrganizationInvitationMapper',
+          OIDCIntegration: '../shared/entities#OIDCIntegration as OIDCIntegrationMapper',
+          User: '../shared/entities#User as UserMapper',
+          SchemaPolicy: '../shared/entities#SchemaPolicy as SchemaPolicyMapper',
+          SchemaPolicyRule: '../shared/entities#SchemaPolicyAvailableRuleObject',
+          SchemaCoordinateUsage: '../shared/mappers#SchemaCoordinateUsageTypeMapper',
+          DocumentCollection: '../shared/entities#DocumentCollection as DocumentCollectionEntity',
+          DocumentCollectionOperation:
+            '../shared/entities#DocumentCollectionOperation as DocumentCollectionOperationEntity',
+          DocumentCollectionConnection: '../shared/entities#PaginatedDocumentCollections',
+          DocumentCollectionOperationsConnection:
+            '../shared/entities#PaginatedDocumentCollectionOperations',
+          FailedSchemaCheck: '../shared/mappers#FailedSchemaCheckMapper',
+          SuccessfulSchemaCheck: '../shared/mappers#SuccessfulSchemaCheckMapper',
+          SchemaPolicyWarningConnection: '../shared/mappers#SchemaPolicyWarningConnectionMapper',
+          Contract: '../shared/mappers#Contract as ContractMapper',
+          ContractConnection: '../modules/schema/providers/contracts#PaginatedContractConnection',
+          ContractCheck:
+            '../modules/schema/providers/contracts#ContractCheck as ContractCheckMapper',
+          ContractVersion:
+            '../modules/schema/providers/contracts#ContractVersion as ContractVersionMapper',
+          BreakingChangeMetadataTarget:
+            '../shared/mappers#BreakingChangeMetadataTarget as BreakingChangeMetadataTargetMapper',
+        },
+      },
+    }),
     // API
     './packages/services/api/src/modules': {
```

Comment:
We still keep this until we finish the migration to avoid blocking features to be developed